src/shared/components/query/StudyListLogic.ts

```diff
@@ -223,10 +223,28 @@ export class FilteredCancerTreeView
 		{
 			let study = node as CancerStudy;
 			let checked = !!this.store.selectedStudyIds.find(id => id == study.studyId);
-			return {checked};
+			let disabled = this.store.isDeletedVirtualStudy(study.studyId);
```

Comment:
i see, we are continuing to show deleted studies, but disabled.  was that a requirement?

Reply:
https://github.com/cBioPortal/cbioportal/issues/3777